src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -1030,6 +1030,9 @@ def forward(
         loss = None
         if labels is not None:
 
+            if labels.max() > self.config.vocab_size:
```

Comment:
Think it should actually be `>=` no? 

If a vocabulary has a vocab_size of 32, then the ids 0 - 31 are used I think. So having a 32 in the labels would fail IMO